### PR TITLE
Created basic object detection using yolo-coco

### PR DESCRIPTION
**Pull request by MicLi01**

### CODE REVIEW

File: src/cv/yolo_video_detect.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import imutils
+import time
+import cv2
+import os
+
+def get_configs(args):
+    # Loads labels
+    labelsPath = os.path.sep.join([args["yolo"], "coco.names"])
```
Comment (chc5) [inline src/cv/yolo_video_detect.py]: Retrieve this file from the environment variable

File: src/cv/yolo_video_detect.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    labels = open(labelsPath).read().strip().split("\n")
+
+    # Create list of colors
+    np.random.seed(42)
```
Comment (chc5) [inline src/cv/yolo_video_detect.py]: Replace this with a time here

File: src/cv/yolo_video_detect.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        start = time.time()
+        layerOutputs = net.forward(ln)
+        end = time.time()
+
+        draw_box(writer, args, start, end, layerOutputs, W, H, frame, colors, labels, total)
+
+    return writer
+
+def draw_box(writer, args, start, end, layerOutputs, W, H, frame, colors, labels, total):
+    # initialize lists of detected bounding boxes, confidences, and class IDs
```
Comment (chc5) [inline src/cv/yolo_video_detect.py]: Put this in a docstring

File: src/cv/yolo_video_detect.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+    # Error occurred while determining # of frames
+    except:
+        print("[INFO] Could not determine # of frames in video")
+        total = -1
+
+    return vs, total
+
+def iterate_frames(args, vs, net, ln, colors, labels, total):
+    writer = None
```
Comment (chc5) [inline src/cv/yolo_video_detect.py]: I recommend that these comments be put into a docstring

File: src/cv/yolo_video_detect.py [hunk 5]
```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import imutils
+import time
+import cv2
+import os
```
Comment (yliu0715) [inline src/cv/yolo_video_detect.py]: I recommend following this import statement ordering: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

File: src/cv/yolo_video_detect.py (hunk 1 above)
Comment (MicLi01) [reply src/cv/yolo_video_detect.py]: changes made

File: src/cv/yolo_video_detect.py (hunk 2 above)
Comment (MicLi01) [reply src/cv/yolo_video_detect.py]: ok

Comment (MicLi01) [reply src/cv/yolo_video_detect.py]: Later on we won't need to do this color generation at all

File: src/cv/yolo_video_detect.py (hunk 5 above)
Comment (MicLi01) [reply src/cv/yolo_video_detect.py]: organized the imports

**Pull request closed without merging**